app/assets/html/sidebar.html

```diff
@@ -18,7 +18,7 @@
                 <div class="conversation-preview menu-item subitem" ng-repeat="conversation in conversationsByUserId[gladiatorId]" ng-show="arenaStateByUserId[gladiatorId]" ng-click="conversationPreviewClicked(conversation.id)">
                     <div class="conversation-heading">
                         <div class="conversation-actions">
-                            <div class="conversation-advance glyphicon glyphicon-arrow-right btn" ng-click="conversationPreviewClicked(conversation.id)"></div>
+                            <div class="conversation-advance glyphicon glyphicon-remove btn" ng-click="deleteIconClicked(conversation.id); $event.stopPropagation();"></div>
```

Comment:
1. Should we make it this easy to delete a conversation?  The `glyphicon-remove` looks similar to the `glyphicon-plus`, and since deleting a conversation isn't supposed to be a common occurrence (i.e. it's done far less than clicking to view a conversation) I'm not sure that enabling one-click deletion from a navigation area will make for good UX...
2.  At an iteration meeting the client noted that he wanted to have an obvious button in place so that even if clicking the `div` opens the conversation, a less experienced user who is looking for a button will find a dummy button and click that.  If we do leave the delete button here, we should also have the view (or whatever you want to call it) button to meet his request since we agreed to it